src/utils/query-category.ts

```diff
@@ -1,9 +1,10 @@
 import type { Category } from "../types/category"
 
 import { API_HOST } from "../constants/env"
+import { SiteKey } from "../types/site"
 
-export async function getCategoryList(): Promise<Category[]> {
-  const response = await fetch(`${API_HOST}/categories`, {
+export async function getCategoryList(site: SiteKey): Promise<Category[]> {
```

Comment:
I'm starting to wonder if at this point we could just make the requests a static return on the FE as this is a "fake" app

Reply:
@npretto I've considered that too, but the same information is also needed for the embedded questions, e.g. we show the products and categories in the dashboard. Having to duplicate the data in both the database and the frontend is a bit iffy, as we have many products.

Reply:
Ah I didn't realize we queried the same db from metabase, all good then, the cache should be enough :)